docs/user-guide.md

```diff
@@ -63,7 +65,8 @@ From the root folder, simply call the `run-tests.ps1` file.
 ```
 
 Server metrics will be output to the `./TestResults` directory. Paging test
-results will found in whatever directory was specified in the `.env` file.
+results will be in a dated sub-directory, e.g.
+`./TestResults/YYYY-mm-dd-HH-MM-SS`.
```

Comment:
```suggestion
`./TestResults/runner/YYYY-mm-dd-HH-MM-SS`.
```
However TestResults folder could change based on PERF_OUTPUT_DIR setting

Reply:
What is documented was actually my intention. But I see now that I missed something and the date folders are underneath `runner`. I preferred keeping those separate, since the PowerShell log files are overwritten every time (currently). Unless you prefer having them in the same parent directory? If you prefer the latter, then I would just get rid of the `runner` portion of the directory path.

Reply:
I think ./TestResults/YYYY-mm-dd-HH-MM-SS is good. PowerShell log files will be overwritten, but dated subdirectory will be still kept.